velox/functions/sparksql/tests/RegexFunctionsTest.cpp

```diff
@@ -529,24 +544,6 @@ TEST_F(RegexFunctionsTest, regexpReplaceMassiveVectors) {
   assertEqualVectors(result, output);
 }
 
-TEST_F(RegexFunctionsTest, regexpReplaceCacheLimitTest) {
```

Comment:
Why are you removing this test?

Reply:
I think is just an artifact of my initial removal of the cache limit. Will add back in push later tonight.